src/app/events/[slug]/components/SponsorLogo.tsx

```diff
@@ -25,13 +20,13 @@ export function SponsorLogo({
   const formattedSponsorTier = formatSponsorTier(tier)
 
   return (
-    <div className={clsx('relative w-full', logoImageConfig.containerStyles)}>
+    <div className={clsx('relative', logoImageConfig.containerStyles)}>
       <Image
         fill
         src={sponsor.image.src}
         alt={`${sponsor.name} Logo - ${formattedSponsorTier} sponsor`}
         sizes={logoImageConfig.imageSizes}
-        className="object-contain object-left-bottom"
+        className="object-contain"
       />
       {sponsor.website && (
         <a
```

Comment:
Just wondering why we have `target="_blank"` here? We usually don't. Just a question.

Love the link as an absolute top layer :)

Reply:
@CharlyMartin is right - Will remove `target="_blank"`  which brings me to [Create External Link Component](https://www.notion.so/filecoin/FF-Create-ExternalLink-component-1307631f282580109a6fef9e98f174eb?pvs=4)

Reply:
Pushed a CR!